garage/tf/algos/ddpg.py

```diff
@@ -42,28 +44,58 @@ def __init__(self,
                  discount=0.99,
                  max_action=None,
                  name=None,
-                 **kwargs):
+                 n_epoch_cycles=20,
+                 max_path_length=None,
+                 n_train_steps=50,
+                 buffer_batch_size=64,
+                 min_buffer_size=1e4,
+                 rollout_batch_size=1,
+                 reward_scale=1.,
+                 input_include_goal=False,
+                 smooth_return=True,
+                 exploration_strategy=None):
         """
         Construct class.
```

Comment:
docstrings for the constructor go in the class docstring, per CONTRIBUTING.md

Reply:
I'll also move constructor docstring to class docstring for DDPG.